packages/tempo/src/plugins/tempo.plugin.ts

```
@@ -152,12 +148,9 @@ export function resolveTermAnchor(tempo: Tempo, terms: TermPlugin[], name: strin
 				case 'start':
 					return start;
 
-				case 'mid': {
-					const midNano = (start.epochNanoseconds + end.epochNanoseconds) / 2n;
-					return new Temporal.ZonedDateTime(midNano, start.timeZoneId, start.calendarId);
-					// const diff = Number(midNano - start.epochNanoseconds);
-					// return start.add({ nanoseconds: diff });
-				}
+				case 'mid':
+					const duration = start.until(end);
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)